apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-spring5/src/main/java/co/elastic/apm/agent/springwebflux/WebfluxHelper.java

```diff
@@ -197,7 +199,11 @@ public static void setTransactionName(@Nullable Transaction<?> transaction, Serv
                 path = exchange.getRequest().getPath().value();
             }
         }
-        String method = exchange.getRequest().getMethodValue();
+        String method = "UNKNOWN";
```

Comment:
[minor] We already have a few occurences of `<unknown>` or `unknown` in lower-case in the apm specs, thus at least using a lower-case variant seems a bit more consistent.